hazelcast/src/test/java/com/hazelcast/test/backup/TestBackupUtils.java

```diff
@@ -142,4 +143,23 @@ public void run() {
             }
         });
     }
+
+    public static <K, V> void assertExpirationTimeExistsEventually(final K key, final BackupAccessor<K, V> accessor) {
+
+        checkInstanceOf(CacheBackupAccessor.class, accessor, "Need to supply a CacheBackupAccessor");
+        assertTrueEventually(new AssertTask() {
+            @Override
+            public void run() {
+                assertTrueEventually(new AssertTask() {
```

Comment:
this `assertTrueEventually` is inside another `assertTrueEventually`. We can remove the one inside.

Reply:
eventual inception... fixed